stats/gRPC.md

```diff
@@ -86,8 +86,8 @@ All server metrics should be tagged with the following.
 
 | Tag name           | Description                                                                                                    |
 |--------------------|----------------------------------------------------------------------------------------------------------------|
-| grpc.server_method | Full gRPC method name, including package, service and method, e.g. com.exampleapi.v4.BookshelfService/Checkout |
-| grpc.server_status | gRPC server status code returned, e.g. OK, CANCELLED, DEADLINE_EXCEEDED                                        |
+| grpc_server_method | Full gRPC method name, including package, service and method, e.g. com.exampleapi.v4.BookshelfService/Checkout |
+| grpc_server_status | gRPC server status code returned, e.g. OK, CANCELLED, DEADLINE_EXCEEDED                                        |
 
 `grpc.server_method` is available in the context for the entire RPC call handling. 
```

Comment:
nit: if all underscores is the way (which may be fine), you'll want to global replace.  I made the same mistake recently in another project and someone had to come back with another commit.

Reply:
done.